zerver/actions/message_edit.py

```diff
@@ -385,6 +385,34 @@ def get_visibility_policy_after_merge(
     return UserTopic.VisibilityPolicy.INHERIT
 
 
+def apply_automatic_unmute_follow_topics_policy(
+    user_profile: UserProfile,
+    message: Message,
```

Comment:
* We are not using the `user_profile` parameter in the function at all.
* Also, we are using only the `sender`, so we can pass that directly instead of `message`.